src/views/payments/Payments.vue

```diff
@@ -91,12 +139,37 @@ export default {
       getProducts(sortBy);
     };
 
+    const paginationCallback = (page) => {
+      const newPagination = {
+        ...pagination.value,
+        currentPage: page - 1,
+      };
+      const transDataList = slicePage({
+        ...newPagination,
+      });
+      dataList.value = [];

```

Comment:
Bai, we'll start practicing good naming conventions. So as an example this is a payment listing page, so we will replace
"transDataList" with "paymentDataList"

Please see codes from line 152 to 165